pymodm/fields.py

```diff
@@ -1056,8 +1058,8 @@ def to_python(self, value):
 # RelatedModelField types.
 #
 
-class EmbeddedDocumentField(RelatedEmbeddedModelFieldsBase):
-    """A field that stores a document inside another document."""
+class _EmbeddedDocumentFieldBase(RelatedEmbeddedModelFieldsBase):
```

Comment:
Since we're trying to refer to objects as "Models" and not "Documents" can this be "_EmbeddedModelFieldBase"?